e2e/e2e-script.sh

```diff
@@ -3,33 +3,32 @@ set -euxo pipefail
 source e2e-helper.sh
 echo "Starting e2e tests"
 
-SUBSCRIPTION_ID="8ecadfc9-d1a3-4ea4-b844-0d9f87e4d7c8" #Azure Container Service - Test Subscription
-RESOURCE_GROUP_NAME="agentbaker-e2e-tests"
-LOCATION="eastus"
-CLUSTER_NAME="agentbaker-e2e-test-cluster"
-
-# Clear the kube/config file for any conflicts
-truncate -s 0 ~/.kube/config
+: "${SUBSCRIPTION_ID:=8ecadfc9-d1a3-4ea4-b844-0d9f87e4d7c8}" #Azure Container Service - Test Subscription
+: "${RESOURCE_GROUP_NAME:=agentbaker-e2e-tests}"
+: "${LOCATION:=eastus}"
+: "${CLUSTER_NAME:=agentbaker-e2e-test-cluster}"
 
 # Create a resource group for the cluster
-if [ $(az group exists -n $RESOURCE_GROUP_NAME --subscription $SUBSCRIPTION_ID) == "false" ]; then
+out=$(az group exists -n $RESOURCE_GROUP_NAME --subscription $SUBSCRIPTION_ID -ojson)
```

Comment:
I think you can unconditionally create the group since it should return 200 either way, but this works for me

Reply:
Ah, I think you're right - creating the group does seem idempotent. I'll remove the conditional :+1: